### PR TITLE
HPCC-25442 Remove/rename misleading variables in ecl-test.json

### PR DESCRIPTION
**Pull request by AttilaVamos**
Remove misleading and not used "defaultSetupClusters" and "defaultTargetClusters" from ecl-test.json

Rename "defaultSetupEngines" to "defaultSetupTargets"

Rename "defaultEngines" to "defaultTargets"

Signed-off-by: Attila Vamos <attila.vamos@gmail.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: testing/regress/ecl-test-cluster160.json [hunk 1]
```diff
@@ -57,12 +48,17 @@
             "soapcall_multihttpheader.ecl:TargetIP=."
         ],
         "engineParams":[
-            "failOnLeaks",
-            "disableLocalOptimizations"
```
Comment (jakesmith) [inline testing/regress/ecl-test-cluster160.json:61]: @AttilaVamos - I assume removed because no longer applicable/used?

**Review by jakesmith — COMMENTED** [review]
@AttilaVamos  one minor question, and PR tick-boxes need filling.

File: testing/regress/ecl-test-cluster160.json (hunk 1 above)
Comment (AttilaVamos) [reply testing/regress/ecl-test-cluster160.json:61]: This is an example configuration file only. That parameter doesn't exists in the default ecl-test.json file and to keep consistency I removed it from here as well.

**Comment by AttilaVamos** [comment]
@jakesmith boxes are ticked now.

**Review by jakesmith — APPROVED** [review]
@AttilaVamos - looks good.

**Comment by AttilaVamos** [comment]
@richardkchapman please merge.

File: testing/regress/ecl-test-cluster160.json (hunk 1 above)
Comment (richardkchapman) [reply testing/regress/ecl-test-cluster160.json:61]: But AFAIK they SHOULD be set on the default ecl-test.json and any time we run regression suite...

Comment (AttilaVamos) [reply testing/regress/ecl-test-cluster160.json]: Yes, it was a mistake to remove. Restored.

**Comment by AttilaVamos** [comment]
@richardkchapman "disableLocalOptimizations" restored, please re-check and merge.

**Pull request merged**